Philips.CodeAnalysis.MaintainabilityAnalyzers/Philips.CodeAnalysis.MaintainabilityAnalyzers.md

```diff
@@ -42,7 +42,7 @@
 | PH2087  | Do not use spaces in filenames               | Not all tools handle files correctly that have spaces in their names. Homebrew scripts are prone to this for example.|
 | PH2088  | Paths should not exceed 260 characters       | This prevents your code to run correctly on older Windows version.|
 | PH2089  | Avoid assignments in conditions              | Conditions shouldn't have any side effects. Assign a variable outside of the condition instead.|
-| PH2090  |	Log Exception                                | Log any of the caught exceptions. The Log method names are configurable in the .editorconfig using key: 'dotnet_code_quality.PH2090.log_method_names'.|
+| PH2090  |	Log Exception                                | Log any of the caught exceptions. The Log method names are configurable by adding an <AdditionalFiles> named AllowedLogMethods.txt.|
```

Comment:
I guess this is a breaking change? Can it support both models? (Static Methods/Classes is a long list, but the log methods is traditionally very small, and would always exist (per client's design), hence baked into .editorconfig is advantageous. Wheres static classes etc is really a meant-to-be temporary list of exceptions.

Reply:
`.editorconfig` can handle up to 5 entries on one line until it becomes a mess. This PR is meant to align the format, so that we can document it once. Hence re-using the AllowedSymbols class here.

Reply:
`LogInfo`, `LogVerbose`, `LogDebug`, `LogError` and `LogWarning` are already 5 entries

Reply:
We could add a default entry: `*.Log.*` to allow all methods on a Type called `Log` in any namespace.

Reply:
> `LogInfo`, `LogVerbose`, `LogDebug`, `LogError` and `LogWarning` are already 5 entries

With the new formatting options you're building (e.g., wildcards), this could be reduced.

Reply:
The pre-baked log exception sounds reasonable. But are you saying if that doesn't fit the client's need, they'd need a permanent file for the allowed log methods?  The breaking change also isn't ideal. I'm thinking that requiring permanent exception files seems like a tall order. Ideally the generic class would support either mechanism.

Reply:
I agree, supporting both mechanisms is best here. This would also eliminate the breaking change.